Firestore/core/src/core/view.cc

```diff
@@ -275,8 +281,14 @@ ViewChange View::ApplyChanges(
       });
 
   ApplyTargetChange(target_change);
-  std::vector<LimboDocumentChange> limbo_changes = UpdateLimboDocuments();
-  bool synced = limbo_documents_.empty() && current_;
+  std::vector<LimboDocumentChange> limbo_changes =
+      targetIsPendingReset ? std::vector<LimboDocumentChange>{}
```

Comment:
Optional: You _may_ be able to replace `std::vector<LimboDocumentChange>{}` with just `{}` to make the code more succinct.

Reply:
It is forbidden with the message `Initializer list cannot be used on the right hand side of operator '?'`